File: 2-programming101/pf_medium_2.py
try:
    bill = float(input("How much was your total bill?\n$ "))
    service = input("How was your serivce today?\nPlease choose: Good, Fair, or Bad\n")
    service_u = service.upper()
    if service_u == "GOOD":
        tip_multiplier = .2
    elif service_u == "FAIR":
        tip_multiplier = .15
    elif service_u == "BAD":
        tip_multiplier = .1
    else:
        print("Please choose the level of service using the terms: Good, Fair, or Bad.\nThanks!")
        exit()
    parts = int(input("How many ways would you like to split the bill?\n"))
    tip_f = bill * tip_multiplier
    tip = "${:,.2f}".format(tip_f)
    total = "${:,.2f}".format(bill + tip_f)
    total_pp = "${:,.2f}".format((bill + tip_f)/parts)
    print(f"Tip amount: {tip}")
    print(f"Total Amount: {total}")
    print(f"Amount per person: {total_pp}")
except ValueError:
    print("Your bill and ways to split it should be a number.")

File: 5_javascript_101/pf_medium_2.js
// Tip Calculator
// Write a function tipAmount that is given the bill amount and the level of service (one of good, fair and poor) and returns the dollar amount for the tip. Based on:

// good -> 20% fair -> 15% bad -> 10%

// tipAmount(100, 'good')
// 20
// tipAmount(40, 'fair')
// 6

function tipAmount(bill, serviceLevel) {
  let tip = 0;
  if (serviceLevel == "good") tip = 0.2;
  else if (serviceLevel == "fair") tip = 0.15;
  else if (serviceLevel == "poor") tip = 0.1;
  return tip * bill;
}
console.log(`$${tipAmount(10, "good")}`);
